src/TTextEdit.cpp

```diff
@@ -1733,6 +1733,11 @@ void TTextEdit::wheelEvent(QWheelEvent* e)
     int yDelta = qRound(delta.y() / 15.0);
     // Store the (rounded) remainder
     mMouseWheelRemainder = QPoint(delta.x() - (15 * xDelta), delta.y() - (15 * yDelta));
+    
+    // TIM DEMO NONSENSE...
+    yDelta = qRound(delta.y() / 10.0); // try smaller increments, so movement 2/3 of original distance needed to move one line, aka 150% speed
+    mMouseWheelRemainder = QPoint(delta.x() - (10 * xDelta), delta.y() - (15 * yDelta));
```

Comment:
I think you are confusing the issue with a second set of calculations here - if nothing else I think you are clobbering the horizontal scrolling as well and I haven't even finished getting that working again yet!

If you want to adjust some values why not doctor the original ones in lines 1733 AND 1735? As it is I think you would be better off changing the `1.0` value in line 1728. That value equates to one line of text scrolled per click (withOUT the `<Ctrl>` key down) - so `3.0` would return to the original 3 lines that @Kebap seems to want back. If you wanted 150% you could try `1.5` though I think that would mean that one click might scroll one line and then the next line would scroll TWO lines and so on - alternating one / two / one / two ... which I do not think *anyone* will like... 🤷‍♂️